ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.4"
+  use_build_cache: true
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 16Gi
+    requests:
+      cpu: "2"
+      memory: 4Gi
+tests:
+- as: sync-upstream-envoy-1.22
```

Comment:
Instead of using the upstream branch, I'd use ours, and would call this job `sync-upstream-envoy-2.3`